src/vars/defaults.yml

```diff
@@ -1,3 +1,4 @@
 ---
 certificate_source: default
 database_mode: internal
+deployment_mode: medium
```

Comment:
this needs to be exposed in metadata for `deploy` and `checks` (probably again with a dedicated `_tuning_mode` file, like we have for `database_mode` etc).

Also, I think this should be called `tuning`, not `deployment`.

@ehelms @ekohl WDYT?

Reply:
It will be a requirement for https://github.com/theforeman/foremanctl/issues/51

As today we don't do anything with those numbers, I wouldn't block this PR on it.

(I think I'd still call the variable `tuning` or `tuning_profile` and keep only one `default.yml` as the profile, we can then add the others once we have the code to actually deploy different configurations)